ui-tests/package.json

```diff
@@ -16,6 +16,7 @@
     "cypress": "^7.6.0",
     "cypress-file-upload": "^5.0.8",
     "cypress-keycloak-commands": "^1.2.0",
+    "cypress-real-events": "^1.5.1",
```

Comment:
what is this library for?

Reply:
this is for arrow key presses and text highlights for hypertext scenario test (`rich text editor` activated). the only text selection shortcuts for cypress was ever introduced was `{selectall}`. I wanted to do multiple text selections as what the test suggested so this is what most people suggested in that kind of scenario. See `Hypertext > try to add several internal link` scenario test for the actual usage.

Reply:
👍🏼